front/app/containers/IdeasShow/GoBackButton.tsx

```diff
@@ -43,7 +52,12 @@ const GoBackButton = memo(({ projectId, className, insideModal }: Props) => {
         padding="0"
         textDecorationHover="underline"
       >
-        {localize(project.attributes.title_multiloc)}
+        <Box display={showButtonText ? 'none' : 'block'} aria-hidden>
```

Comment:
This is inside a button so please add `as="span"` to the `Box`. Otherwise, it will be a `div` and this is not valid.